.env.codeception

```diff
@@ -0,0 +1,10 @@
+TEST_SITE_DB_NAME="planet4_dev"
+TEST_SITE_DB_HOST="db"
+TEST_SITE_DB_USER="root"
```

Comment:
I would prefer if we put those (db_user and db_pass) as circleCI project variables